docs/API.md

```
@@ -1,21 +1,929 @@
+
+---
+
+## Events (SSE)
+
+### `GET /api/events`
+
+Server-Sent Events stream for real-time push updates. Compression is **disabled** on this route to prevent chunk buffering.
+
+**Headers sent by server:**
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)